simpleocr/files.py

```diff
@@ -28,6 +29,18 @@ def try_extensions(path, extensions):
     return None
 
 
+def find_file(file_name):
```

Comment:
The behaviour didn't change for me before and after this PR (before your fix) - creating a ImageFile with a absolute path works, but it fails with a relative path. 
Are you testing this on windows? Could the behaviour be different?

I've created #25 to track this, so we don't lose the context later